master/internal/db/postgres_users.go

```diff
@@ -317,31 +317,18 @@ func (db *PgDB) AuthTokenKeypair() (*model.AuthTokenKeypair, error) {
 }
 
 // UpdateUserSetting updates user setting.
-func UpdateUserSetting(setting *model.UserWebSetting) error {
-	if len(setting.Value) == 0 {
-		_, err := Bun().NewDelete().Model(setting).Where(
-			"user_id = ?", setting.UserID).Where(
-			"storage_path = ?", setting.StoragePath).Where(
-			"key = ?", setting.Key).Exec(context.TODO())
-		return err
-	}
-
-	_, err := Bun().NewInsert().Model(setting).On("CONFLICT (user_id, key, storage_path) DO UPDATE").
-		Set("value = EXCLUDED.value").Exec(context.TODO())
-	return err
-}
-
-// OverwriteUserSetting resets user settings and fills them with passed values.
-func OverwriteUserSetting(userID model.UserID, settings []*userv1.UserWebSetting) error {
```

Comment:
question to better follow these changes: what was the previous use-case for this?

Reply:
This endpoint was added and removed in this PR after discussion with Claire.